mcs/class/System.Drawing/System.Drawing/SR.cs

```diff
@@ -0,0 +1,72 @@
+//
```

Comment:
We generate this file, look how it's done in e.g. mcs/class/System.Core/Makefile